nbformat/tests/test_sign.py

```diff
@@ -28,7 +29,22 @@ def setUp(self):
     
     def tearDown(self):
         shutil.rmtree(self.data_dir)
-    
+   
+    def test_invalid_db_file(self):
+        invalid_sql_file = os.path.join(self.data_dir, 'invalid_db_file.db')
+        with open(invalid_sql_file, 'w') as tempfile:
+            tempfile.write(u'[invalid data]')
+
+        invalid_notary = sign.NotebookNotary(
+            db_file=invalid_sql_file,
+            secret=b'secret',
+            data_dir='.',
+        )
+        invalid_notary.sign(self.nb)
+
+        self.assertIn('invalid_db_file.db', os.listdir(self.data_dir))
+        self.assertIn('invalid_db_file.db.bak', os.listdir(self.data_dir))
```

Comment:
I think we already have a dependency on testpath for the tests, so you can use [testpath.assert_isfile()](http://testpath.readthedocs.org/en/latest/fsasserts.html#testpath.assert_isfile) to check these.